README.md

```diff
@@ -0,0 +1,10 @@
+<<<<<<< ci/basic-setup

```

Comment:
Looks like something happened during a merge... these comments should be removed